lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -265,12 +265,12 @@ <h3 class="hd hd-3">${_("Coupon Code List")}</h3>
              url: "${section_data['sale_validation_url'] | n, js_escaped_string}",
              success: function (data) {
                  $('#error-msg').attr('class','success-msgs')
-                 $('#error-msg').html(data.message).show();
+                 $('#error-msg').text(data.message).show();
```

Comment:
These all look good, but you'll need to test to be sure that data.message does not include any HTML tags.  If it does, you'll need to use HtmlUtils.  Same with the changes below.